IN-CORE/incore-playbook
# Fix style of tornado and others

To test:
1. remove map.config.js, global.config.js
2. copy map.config.joplin.js to be map.config.js
3. copy global.config.joplin.js to be global.config.js
4. npm run start:dev

After it runs, 
1. login
2. go to 3-3
3. display earthquake and tornado to see if they have correct style

The style for earthquake has been updated in both dev and prod geoserver
